scenes/scene.yaml

```diff
@@ -80,10 +88,11 @@ styles:
 sources:
     osm:
         type: MVT
-        url:  https://vector.mapzen.com/osm/all/{z}/{x}/{y}.mvt
+        #url:  https://vector.mapzen.com/osm/all/{z}/{x}/{y}.json
+        mbtiles: nyc.mbtiles
```

Comment:
Yes, you'll need to track that a mbtiles file has the same source if you want to avoid writing garbage to some other mbtiles file.